doc/events.md

```diff
@@ -50,6 +50,18 @@ class PreSignSubscriber implements EventSubscriberInterface
 }
 ```
 
+The `PreSignEvent` also lets you override the default signer name/email dynamically:
+
+```php
+public function preSign(PreSignEvent $preSign)
+{
+    $envelopeBuilder = $preSign->getEnvelopeBuilder();
+
+    $envelopeBuilder->setDefaultSignerName('John DOE');
+    $envelopeBuilder->setDefaultSignerEmail('john.doe@example.com');
```

Comment:
Sorry to borrow more of your time, but could you also add that one could call addCarbonCopy or addSigner in case multiple signers are expected, or to be notified?

Or in fact anything you'd want for programmatically changing the signature request.

Reply:
The request query parameters `signerName` and `signerEmail` don't **add** but **set** values.

Otherwise, I should adjust the query request parameters as string and/or array: if it's a string, call the **setter**; if it's an array, call the **adder**. WDYT @GregoireHebert ?